src/input.cpp

```diff
@@ -925,40 +939,48 @@ void input_context::display_help()
             } else {
                 col = global_key;
             }
-            mvwprintz( w_help, i + 8, 4, col, "%s: ", get_action_name( action_id ).c_str() );
-            mvwprintz( w_help, i + 8, 52, col, "%s", get_desc( action_id ).c_str() );
+            mvwprintz( w_help, i + 10, 4, col, "%s: ", get_action_name( action_id ).c_str() );
+            mvwprintz( w_help, i + 10, 52, col, "%s", get_desc( action_id ).c_str() );
         }
+
+        filter_phrase = string_input_win_from_context( w_help, ctxt, filter_phrase, legwidth - 1, 4, 8, legwidth, false, action,
+                                                       raw_input_char, current_search_cursor_pos, "", -1, -1, true, false,
+                                                       std::map<long, std::function<void()>>(), character_blacklist );
+
+        if ( scroll_offset > filtered_registered_actions.size() )
```

Comment:
Please read doc/CODE_STYLE.txt - then/else statements should always be in a separate block (surrounded by braces) even if it's only one statement.